app/controllers/auth/AuthAction.scala

```diff
@@ -44,9 +45,11 @@ class AuthActionImpl @Inject()(
 
   override def invokeBlock[A](request: Request[A], block: AuthenticatedRequest[A] => Future[Result]): Future[Result] = {
 
+    val compositePredicate = CompositePredicate(CredentialStrength(CredentialStrength.strong), ConfidenceLevel.L200)
```

Comment:
You can use `and` instead of construcing a composite predicate  
```suggestion
    val compositePredicate = CredentialStrength(CredentialStrength.strong) and ConfidenceLevel.L200
```